File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/filtri/filtroPC_Finalita_Rer.ts
import { Regolamenti } from "../metaschema/Regolamenti";
import { GruppoFinalita } from "../metaschema/utilizzi/GruppoFinalita";
import { Specie } from "../metaschema/utilizzi/Specie";

export class FiltroPC_Finalita_Rer{
    regolamento: Regolamenti;
    specie: Specie;
    finalita: GruppoFinalita;
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreDTOStd/InData/Metaschema/FiltroPC_Finalita_Rer.cs
using AgronicaCoreModelsSTD.metaschema;
using AgronicaCoreModelsSTD.metaschema.utilizzi;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreDTOStd.InData.Metaschema
{
    public class FiltroPC_Finalita_Rer
    {
        public Regolamenti regolamento { get; set; }

        public Specie specie { get; set; }

        public GruppoFinalita finalita { get; set; }
    }
}
